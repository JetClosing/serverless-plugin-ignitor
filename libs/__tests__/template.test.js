const template = require('../template');

test('function exported', () => {
  expect(template.generate).toBeDefined();
});

test('generates expected output', () => {
  expect(template.generate('foo', 'handler')).toMatchSnapshot();
});