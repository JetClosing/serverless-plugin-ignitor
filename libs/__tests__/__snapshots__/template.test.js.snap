// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generates expected output 1`] = `
"const original = require('../foo');

module.exports['handler'] = function (evt, ctx, cb) {
  if (evt.source === 'aws.events') {
    return cb(null, 'pinged');
  }

  return original.handler(evt, ctx, cb);
};"
`;
